game-sales-tracker/src/app/authorization-module/login-form/login-form.component.ts

```diff
@@ -36,7 +40,13 @@ export class LoginFormComponent implements OnInit {
     }
 
     public onSubmit(): void {
-        this.credentials = this.loginForm.value;
-        this.loginFormDirective.resetForm();
+        this.authService.login(this.loginForm).subscribe(
+            () => this.router.navigate(["/"]).then(
+                this.loginFormDirective.resetForm()),
```

Comment:
Why not?
   () => {
        this.router.navigate(["/"]);
        this.loginFormDirective.resetForm();
}

Reply:
Why not?
() => this.router.navigate(["/"]).then(
                this.loginFormDirective.resetForm()),